types/tabulator-tables/index.d.ts

```diff
@@ -122,7 +122,7 @@ declare namespace Tabulator {
         /**  Setting this option to true will cause Tabulator to create a list of page size options, that are multiples of the current page size. In the example below, the list will have the values of 5, 10, 15 and 20.
 
     When using the page size selector like this, if you use the setPageSize function to set the page size to a value not in the list, the list will be regenerated using the new page size as the starting valuer	*/
-        paginationSizeSelector?: true | number[];
+        paginationSizeSelector?: true | false | number[];
```

Comment:
The docs don't say anything about setting this to false. It is default false and redundant to set.

http://tabulator.info/docs/4.2/release#page